draft-ietf-ppm-dap.md

```diff
@@ -789,26 +791,21 @@ include any information specifying a particular batch (see {{query}}). The
 Leader selects a recent batch to aggregate. The Leader MUST select a batch that
 has not yet been associated with a collection job.
 
-In addition to the minimum batch size common to all query types, the
-configuration may include a parameter `max_batch_size` that determines maximum
-number of reports per batch.
-
-If the configuration does not include `max_batch_size`, then the Aggregators
-can output any batch size that is larger than or equal to `min_batch_size`.
-This is useful for applications that are not concerned with sample size, i.e.,
-the privacy guarantee is not affected by the sampling rate of the population,
-therefore a larger than expected batch size does not weaken the designed privacy
-guarantee.
+The Aggregators can output batches of any size that is larger than or equal to
+`min_batch_size`. This is useful for applications that are not concerned with
+sample size, i.e., the privacy guarantee is not affected by the sampling rate of
+the population, therefore a larger than expected batch size does not weaken the
+designed privacy guarantee.
```

Comment:
This text now lacks important. There is only a minimum batch size, hence no application of DAP is allowed to be "concerned with the sample size".

I think what we want to do now instead is say something generic about how privacy guarantees of DAP depend on the batch size, perhaps when composed with some DP policy.

However I wonder if we still think this is accurate. My intuition is that choosing a larger batch size strictly increases privacy, regardless of how DP is implemented. (Though utility may be negatively impacted.) Is this correct?

cc/ @wangshan since according to `git blame`, you wrote this text originally.

Reply:
generally speaking larger batch size gives better DP guarantee, however, there can be a case where the DP guarantee is based on a certain sampling rate and allowing too big a batch may break that condition. But in practice, I don't think sampling would be implemented in a way that make this a problem. If we decide to remove max_batch_size, I agree it doesn't make much sense to mention sampling here.

Reply:
I got rid of this text and merged the next paragraph into it, since that paragraph is also concerned with the range of possible batch sizes. The following paragraph is concerned with the batch sizes that might be generated by an implementation, but IMO the existing text makes it clear the difference between specification & implementation advice (and this section is not normative).